conda_builder_linux/Dockerfile.cuda

```diff
@@ -0,0 +1,32 @@
+# This is a clone of ./Dockerfile, but layered over a different base image
+# to include a CUDA 7.5 development environment.
+
+FROM nvidia/cuda:7.5-devel-centos7
```

Comment:
Please base this off of their 7.5-devel-centos6 image instead, for wider output compatibility.